product-service/consts/stock-db-queries.ts

```diff
@@ -0,0 +1 @@
+export const insertStock = (productId, count) => `insert into stocks (product_id, count) values (${productId}, ${count})`;
```

Comment:
Consider using SQL query params instead of inserting data directly into the query to avoid SQL Injections.